ods_ci/tests/Tests/400__ods_dashboard/415__ods_dashboard_projects/415__ods_dashboard_projects_notebook_images.robot

```diff
@@ -0,0 +1,63 @@
+*** Settings ***
+Documentation      Suite to test the spawn of different notebook images
+Library            SeleniumLibrary
+Library            OpenShiftLibrary
+Resource           ../../../Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/Projects.resource
+Resource           ../../../Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/Workbenches.resource
+Resource           ../../../Resources/RHOSi.resource
+Suite Setup        Project Suite Setup
+Suite Teardown     Project Suite Teardown
+
+
+*** Variables ***
+${PRJ_TITLE}    ODS-CI DS Project Notebook Images
+${PRJ_RESOURCE_NAME}    notebook-images-ds-project
+${PRJ_DESCRIPTION}    ${PRJ_TITLE} is a test project for validating notebook images and shared by multiple tests
+${INTEL_AIKIT_APPNAME}          aikit
+${INTEL_AIKIT_OPERATOR_NAME}    Intel® oneAPI AI Analytics Toolkit Operator
+${IMG_NAME}   oneAPI AI Analytics Toolkit
+${WORKBENCH_TITLE}    aikitwb
+${PV_DESCRIPTION}    PV for AiKit workbench
+
+
+*** Test Cases ***
+Verify user can create a workbench using Intel AiKit image
+    [Documentation]    Verifies that a workbench can be created using Intel AiKit image
+    [Tags]    ODS-2173    Tier2
+    Check And Install Operator in Openshift    ${INTEL_AIKIT_OPERATOR_NAME}    ${INTEL_AIKIT_APPNAME}
+    Create Tabname Instance For Installed Operator        ${INTEL_AIKIT_OPERATOR_NAME}
+    ...    AIKitContainer    ${APPLICATIONS_NAMESPACE}
+    Go To RHODS Dashboard
+    Open Data Science Project Details Page       project_title=${PRJ_TITLE}
+    Create Workbench    workbench_title=${WORKBENCH_TITLE}  workbench_description=workbench for testing
+    ...        prj_title=${PRJ_TITLE}    image_name=${IMG_NAME}  version=${NONE}    deployment_size=Small
+    ...        storage=Persistent  pv_name=aikitpv  pv_existent=${FALSE}
+    ...        pv_description=${PV_DESCRIPTION}  pv_size=1
+    ...        press_cancel=${FALSE}    envs=${NONE}
+    Wait Until Workbench Is Started     workbench_title=${WORKBENCH_TITLE}
+    Open Data Science Projects Home Page
+    Wait Until Project Is Listed    project_title=${PRJ_TITLE}
+    Launch And Access Workbench From Projects Home Page    workbench_title=${WORKBENCH_TITLE}
+    ...    project_title=${PRJ_TITLE}    username=${TEST_USER_3.USERNAME}
+    ...    password=${TEST_USER_3.PASSWORD}    auth_type=${TEST_USER_3.AUTH_TYPE}
+
+
+*** Keywords ***
+Project Suite Setup
+    [Documentation]    Suite setup steps for testing DS Projects. It creates some test variables
+    ...                and runs RHOSi setup
+    Set Library Search Order    SeleniumLibrary
+    ${to_delete}=    Create List    ${PRJ_TITLE}
+    Set Suite Variable    ${PROJECTS_TO_DELETE}    ${to_delete}
```

Comment:
Another option, I think easier to understand, could be to declare ${PROJECTS_TO_DELETE} as empty list in the suite's Variables section, and then use `Append To List` after the project is created